src/Coders.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <string>
+#include <iostream>
+
+
+class Coders
+{
+private:
+    std::string userName;
+    std::string discordNickname;
+    std::string githubLink;
+    std::string firecodeLink;
+public: 
+    enum class Group {weekends, evenings};
+    Group SetGroup;
```

Comment:
Why the group member field is public? The enum itself (line above) can be in a public scope. This field should rather be in a private scope.